tensorflow/lite/micro/tools/ci_build/test_code_style.sh

```diff
@@ -31,24 +31,15 @@ make -f tensorflow/lite/micro/tools/make/Makefile third_party_downloads
 # scripts fail with an error code.
 set +e
 
-# The pigweed scripts only work from a git repository and the Tensorflow CI
```

Comment:
Should this be part of the same change?

Reply:
created https://github.com/tensorflow/tflite-micro/pull/173. I'll fix up the current PR once #173 is merged.